boxmot_ros_node:
  ros__parameters:
    tracking_model: "deepocsort"
    reid_model: "osnet_x0_25_msmt17.pt"
    input_topic: "/yolo_ros/detection_result"
    publish_annotated_image: True
    annotated_topic: "/boxmot_ros/annotated_image"
    detailed_topic: "/boxmot_ros/tracking_result"
    threshold: 0.25
    device: "0"
